docs/content/self-hosting/self-hosting.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Self-Hosting Aperture
+sidebar_label: Self-Hosting
+sidebar_position: 6.5
+keywords:
+  - self-hosted
+---
+
+Usually, Aperture Agents are the only required Aperture component to be
+installed. In such a setup, Agents are connecting to [FluxNinja ARC][].
+
+If you want to have control over the infrastructure and data, it's also possible
+to self-host your own Aperture Controller. In such a setup, Agents and
+Controller comprise a fully functional Aperture system.
+
+Note that [FluxNinja ARC][] can integrate with self-hosted controller too,
+providing a visual way to manage policies and view the infrastructure, along
```

Comment:
instead of a visual way, we can say `providing an easy way to manage policies and holistic view of the infrastructure`